sphinx/quickstart.py

```diff
@@ -213,9 +213,10 @@
 # directly to the root of the documentation.
 #html_extra_path = []
 
-# If not '', a 'Last updated on:' timestamp is inserted at every page bottom,
-# using the given strftime format.
-#html_last_updated_fmt = '%%b %%d, %%Y'
+# If not None, a 'Last updated on:' timestamp is inserted at every page
+# bottom, using the given strftime format.
+# The empty string is equivalent to '%b %d, %Y'.
```

Comment:
Ah, OK. I should have simply copied the %% from the original code. But, "make check" yields
"No errors found." with my current version. So what tests am I supposed to run? And how? (I'm on ubuntu 14.10, python 2.7.8.)